conf/masters

```diff
@@ -11,11 +11,10 @@
 
 # An Alluxio master will be started on each of the machines listed below.
 #
-# In HA mode, Alluxio will use Zookeeper to decide which of the masters should
-# act as the primary. In non-HA mode, the master started on the host identified
-# by the first entry in this file will act as the primary; the remaining hosts
+# In HA mode, Alluxio will use Embedded Journal or Zookeeper to decide which of
```

Comment:
Do we really want to refer to the "Embedded Journal" as the mechanism which determines the Alluxio master in HA mode? I feel like "Embedded Journal" describes the Alluxio-only HA feature but not necessarily how it determines the leader.

I think it would be more accurate if we said something to the tune of "Alluxio will either use Zookeeper or, if configured with the embedded journal, an implementation of the [Raft protocol](https://raft.github.io/) to elect a leader"